pkg/storage/sqlcommon/sqlcommon.go

```diff
@@ -326,6 +327,23 @@ func NewDBInfo(db *sql.DB, stbl sq.StatementBuilderType, sqlTime interface{}) *D
 	}
 }
 
+var (
+	emailRegex       = regexp.MustCompile(`^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$`)
+	phoneNumberRegex = regexp.MustCompile(`^(\+\d{1,2}\s?)?1?\-?\(?\d{3}\)?[\s.-]\d{3}[\s.-]\d{4}$`)
+	ssnRegex         = regexp.MustCompile(`^\d{3}-\d{2}-\d{4}$`)
```

Comment:
Personal opinion, this looks too premature of a PII check to me, as it's specific to the USA. Are we going to add PII checks for other types of IDs too?

Reply:
I am also struggling, with what to verify under PII, what could be the other patterns we can compare and check against?